languages/python/README.md

```diff
@@ -37,12 +37,12 @@ Before launch, we need all of the following parts to be completed:
 
 ### Representer
 
-- [ ] Build Representer
+- [x] Build [Representer](https://github.com/exercism/python-representer)
 - [ ] Deploy Representer
 
 ### Test Runner
 
-- [ ] Build Test Runner
+- [x] Build [Test Runner](https://github.com/exercism/python-test-runner)
 - [ ] Deploy Test Runner
```

Comment:
According to [this list](https://github.com/exercism/automated-tests/issues/32). the Python test runner is also working, so I think you can consider it deployed.

Reply:
I'll tick both.

Reply:
Great!